src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -1671,8 +1671,9 @@ def _parse_aoi(data):
     serializer.is_valid(raise_exception=True)
     area_of_interest = serializer.validated_data.get('area_of_interest')
     wkaoi = serializer.validated_data.get('wkaoi')
+    layer_overrides = data.get('layer_overrides', {})
 
-    return area_of_interest, wkaoi
+    return area_of_interest, wkaoi, layer_overrides
```

Comment:
Adding a third return value here breaks all the `analyze` endpoints, which only expect two values:

https://github.com/WikiWatershed/model-my-watershed/blob/9b1aebada97f14480a3fdbab70491ab24e8d8993/src/mmw/apps/geoprocessing_api/views.py#L524

Since `layer_overrides` only makes sense for modeling endpoints and not analyze ones (where analyzing a different layer is a path parameter, e.g. land and streams), we should return it from `_parse_modeling_input`.